packages/nouns-contracts/test/foundry/NounsDAOLogic/NounsDAOVotes.t.sol

```diff
@@ -149,4 +150,44 @@ contract NounsDAOLogicVotes_ActiveState_Test is NounsDAOLogicVotesBaseTest {
         vm.prank(proposer);
         dao.castRefundableVote(proposerTokenIds, proposalId, 1);
     }
+
+    function test_givenTokenTransferAtCurrentBlock_reverts() public {
```

Comment:
let's add more tests here for the use case:
`nounsToken` delegated on the same block